mods/cnc/chrome/settings.yaml

```diff
@@ -1,30 +1,22 @@
 Container@SETTINGS_PANEL:
 	Logic: SettingsLogic
-		HotkeyGroups:
-			Game Commands:
-				Template: TWO_COLUMN
-				Types: OrderGenerator, World, Menu
-			Viewport Commands:
-				Template: TWO_COLUMN
-				Types: Viewport
-			Observer / Replay Commands:
-				Template: TWO_COLUMN
-				Types: Observer, Replay
-			Unit Commands:
-				Template: THREE_COLUMN
-				Types: Unit
-			Unit Stance Commands:
-				Template: TWO_COLUMN
-				Types: Stance
-			Production Commands:
-				Template: THREE_COLUMN
-				Types: Production, ProductionSlot
-			Support Power Commands:
-				Template: THREE_COLUMN
-				Types: SupportPower
-			Music Commands:
-				Template: TWO_COLUMN
-				Types: Music
+		ButtonStride: 120, 0
+		Panels:
+			Display:
```

Comment:
These keys don't seem to be used at all?

How about using a straight dictionary?
```yaml
Panels:
	DISPLAY_PANEL: Display
	AUDIO_PANEL: Audio
	INPUT_PANEL: Input
	HOTKEYS_PANEL: Hotkeys
	ADVANCED_PANEL: Advanced
```
?

Reply:
Yes, this is more concise and straight-forward :+1: